openshift/template.yaml

```diff
@@ -433,6 +433,8 @@ parameters:
   value: "300m"
 - name: PROW_JOBS_SCRAPER_MEMORY_REQUEST
   value: "400Mi"
+- name: GCS_BUCKET_NAME
```

Comment:
I think you're missing a usage of that parameter

Reply:
Sorry my bad